src/lib/y2users/linux/local_reader.rb

```diff
@@ -79,6 +83,14 @@ def read_passwords(config)
           user.password = password
         end
       end
+
+      def read_authorized_keys(config)
```

Comment:
It is ok for now, but I think that `Linux::Reader` and `Linux::LocalReader` deserve a `Linux::BaseReader` class to allocate shared logic like this for reading the authorized keys.

Reply:
Yes, I thought exactly the same few minutes ago, when moving the read of authorized_keys to those readers.